lib/ruby-next/commands/nextify.rb

```diff
@@ -210,6 +236,10 @@ def stdout?
       def single_version?
         single_version || specified_rewriters
       end
+
+      def overwrite_original_file?
+        single_version && overwrite_original_file
```

Comment:
We can remove `single_version` from here, since we explicitly check for it presence after parsing

```suggestion
        overwrite_original_file
```

Reply:
done